packages/react-app/src/components/header.js

```diff
@@ -61,9 +61,9 @@ export default function (props) {
                 <Navbar.Collapse id="basic-navbar-nav">
                     <Nav className="d-flex justify-content-end align-items-center">
 
-                        <Nav.Link href="/new/worker">New Stake</Nav.Link>
-                        <Nav.Link href="/manage">Manage</Nav.Link>
-                        <Nav.Link href="https://www.coinbase.com/earn/nucypher" target="blank">Learn</Nav.Link>
+                        <Nav.Link><Link to="/new/worker">New Stake</Link></Nav.Link>
+                        <Nav.Link><Link to="/manage">Manage</Link></Nav.Link>
+                        <Nav.Link><Link to="https://www.coinbase.com/earn/nucypher" target="blank"></Link>Learn</Nav.Link>
```

Comment:
I thought `<Link>` doesn't work with external links...which is why is why it was `<a>` before?

Reply:
That wasn't the issue.  The issue is that the site is a single page app with its own router. https://reactrouter.com/web/guides/quick-start
All internal links from one page to another need to use the router which is the 
`<Link to="location">`
  
Now that I look at this, it looks like I messed up the _external_ link 🤦

Reply:
I'm working on fixing the issue with why it crashes on external entry like if you just go directly to here:
https://stake.nucypher.network/new/worker/join

Reply:
So just to confirm, if you click on 'Learn' the external link works correctly?

Reply:
Yep I reverted that one.
And now with the build being fixed,  the internal ones _would_ work but it's still better to use router links for those since it will reload the whole page if you use an href="" link (this may depend on different browsers)